python/kvikio/kvikio/__init__.py

```diff
@@ -1,18 +1,18 @@
 # Copyright (c) 2021-2024, NVIDIA CORPORATION. All rights reserved.
 # See file LICENSE for terms.
 
-from ._lib import libkvikio  # type: ignore
-from ._version import __git_commit__, __version__  # noqa: F401
-from .cufile import CuFile  # noqa: F401
+from kvikio._lib import buffer, driver_properties  # type: ignore
+from kvikio._version import __git_commit__, __version__  # noqa: F401
```

Comment:
Rather than noqa these, we can define `__all__` so it is clear that this is meant to be part of the public API.

Reply:
Good point, done